mapbox_vector_tile/encoder.py

```diff
@@ -283,17 +268,16 @@ def _handle_attr(self, layer, feature, props):
                     val = layer.values.add()
                     if isinstance(v, bool):
                         val.bool_value = v
-                    elif (isinstance(v, str)):
+                    elif isinstance(v, str):
                         if PY3:
-                            val.string_value = str(v)
+                            val.string_value = v
                         else:
-                            val.string_value = unicode(v, 'utf8')
-                    elif (isinstance(v, str if PY3 else unicode)):
+                            val.string_value = unicode(v, 'utf-8')
+                    elif isinstance(v, unicode):
                         val.string_value = v
-                    elif (isinstance(v, int)) or (
-                            isinstance(v, int if PY3 else long)):
+                    elif isinstance(v, int) or isinstance(v, long):
                         val.int_value = v
-                    elif (isinstance(v, float)):
+                    elif isinstance(v, float):
```

Comment:
What do you think of spelling the `or isinstance` checks with a tuple instead? For example:

```python
isinstance(v, (str, unicode))
```